lib/hoe/publish.rb

```diff
@@ -100,7 +100,7 @@ def initialize_publish
   # Declare a dependency on rdoc, IF NEEDED.
 
   def activate_publish_deps
-    dependency "rdoc", "~> 4.0", :developer if need_rdoc
+    dependency "rdoc", ">= 4.0", :developer if need_rdoc
```

Comment:
No unbounded deps please.

http://www.zenspider.com/ruby/2008/10/rubygems-how-to-preventing-catastrophe.html